### PR TITLE
fix: madvise_random on not(unix)

### PR DESCRIPTION
**Pull request by kinosang**
should return () instead of Result<()>

### CODE REVIEW

File: src/index.rs [hunk 1]
```diff
@@ -499,7 +499,7 @@ impl IndexTable {
 
 	#[cfg(not(unix))]
 	fn madvise_random(&self, _map: &mut memmap2::MmapMut) {
```
Comment (shamilsan) [inline src/index.rs:501]: The code might be even shorter.

```suggestion
	fn madvise_random(&self, _map: &mut memmap2::MmapMut) {}
```

File: src/index.rs [hunk 2]
```diff
@@ -499,7 +499,7 @@ impl IndexTable {
...
-		Ok(())
+		()
 	}
```
Comment (shamilsan) [inline src/index.rs:503]:
```suggestion
```

**Comment by shamilsan** [comment]
Also waiting for this PR to be merged as it is not possible to build the project against Windows.

**Comment by arkpar** [comment]
published as `0.3.2`

**Pull request merged**